frontend/app/controllers/modal/activity.js

```diff
@@ -32,7 +38,7 @@ export default Ember.Controller.extend({
     get() {
       let from = this.get('model.from')
 
-      return from && from.format(this.dateFormat)
+      return (from && from.format(this.dateFormat)) || moment().format(this.dateFormat)
```

Comment:
No need for parentheses, `&&` has higher precedence than `||` and eslint complains about unnecessary parentheses.